src/MainWindow/HomePage/AdvancedScanOptions.jsx

```diff
@@ -129,9 +166,36 @@ const AdvancedScanOptions = ({
               />
             </div>
           )}
+          <hr />
+          <div id="max-concurrency-toggle-group">
+            <input
+              type="checkbox"
+              id="max-concurrency-toggle"
+              aria-describedby='max-concurrency-tooltip'
+              checked={advancedOptions.maxConcurrency}
+              onFocus={() => handleOnFocus()}
+              onBlur={() => setShowToolTip(false)}
+              onMouseEnter={() => handleOnMouseEnter()}
+              onMouseLeave={() => setIsMouseEvent(false)}
+              onChange={handleSetAdvancedOption(
+                "maxConcurrency",
+                (e) => e.target.checked
+              )}
+            />
+            <label htmlFor="max-concurrency-toggle">
+              Slow Scan Mode
+            </label>
+            <div className="custom-tooltip-container">
+              <ToolTip 
+                description={'Scan 1 page at a time instead of multiple pages concurrently.'} 
+                id='max-concurrency-tooltip'
+                showToolTip={showToolTip}
+              />
+              <img id='max-concurrency-tooltip-img' src={questionMarkIcon} aria-describedby='max-concurrency-tooltip'></img>
```

Comment:
wdyt of making use of the elements `onMouseEnter` and `onMouseLeave` instead? Then you can remove line 33-53
Woops also think you may have missed the alt text, linter is complaining
```suggestion
              <img id='max-concurrency-tooltip-img' src={questionMarkIcon} aria-describedby='max-concurrency-tooltip' onMouseEnter={() => setShowToolTip(true)} onMouseLeave={() => setShowToolTip(false)} alt="tooltip icon for slow scan mode"/>
```